apps/web/components/CardWithFeaturedItems/CardWithFeaturedItems.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  FocusableBox,
+  Hidden,
+  Link,
+  Tag,
+  Text,
+} from '@island.is/island-ui/core'
+import * as styles from './CardWithFeaturedItems.css'
+import { LinkType, useLinkResolver } from '@island.is/web/hooks'
+import { Featured } from '@island.is/web/graphql/schema'
+
+type CardWithFeaturedItemsProps = {
+  heading: string
+  imgSrc: string
+  alt: string
+  href: string | null
+  dataTestId?: string
+  buttonTitle?: string
+  featuredItems: Featured[]
+}
+
+export const CardWithFeaturedItems = ({
+  heading,
+  imgSrc,
+  alt,
+  href,
+  dataTestId,
+  featuredItems,
+  buttonTitle,
+}: CardWithFeaturedItemsProps) => {
+  const { linkResolver } = useLinkResolver()
+  return (
+    <FocusableBox
+      href={href ?? undefined}
+      background="purple100"
+      borderRadius="large"
+      color="purple"
+      data-testid={dataTestId}
+      className={styles.container}
+      justifyContent="spaceBetween"
+      padding={3}
+      flexDirection={['columnReverse', 'row']}
+    >
+      <Box display="flex" flexDirection="column" justifyContent="spaceBetween">
+        <Box>
+          <Text variant="h3" color="purple600" truncate marginBottom={[1, 0]}>
+            {heading}
+          </Text>
+        </Box>
+        <Box height="full">
+          {featuredItems.length > 0 && (
+            <Hidden below="sm">
+              <Box marginY={2}>
+                {featuredItems.map((item: Featured, index: number) => {
+                  const cardUrl = linkResolver(item.thing?.type as LinkType, [
```

Comment:
I'm wondering if it'd be better to filter out items that don't resolve to a link since otherwise items will appear with empty links